src/clients/sponsor.ts

```diff
@@ -0,0 +1,45 @@
+import { AuthType, InstanceOptions, IOClient, IOContext } from '@vtex/api'
+
+export class Sponsor extends IOClient {
+
+  private region: string
+  private account: string
+  private workspace: string
+
+  constructor(context: IOContext, options: InstanceOptions) {
+    super(
+      context,
+      {
+        ...options,
+        authType: AuthType.bearer,
+      }
+    )
+    const { region, account, workspace } = context
+    this.region = region
+    this.account = account
+    this.workspace = workspace
+  }
+
+  public getSponsorAccount = async () =>
+    this.http.get(this.routes.getSponsorAccount, { metric: 'get-sponsor-account' })
+
+  public getEdition = async () =>
+    this.http.get(this.routes.getEdition, { metric: 'get-edition' })
+
+  public setEdition = async (account: string, edition: string) =>
+    this.http.put(this.routes.setEdition(account), { edition }, { metric: 'set-edition' })
+
+  public runHouseKeeper = async () =>
+    this.http.post(this.routes.runHouseKeeper, {}, { metric: 'run-house-keeper' })
+
+  private get routes() {
+    return {
+      getSponsorAccount: `http://kube-router.${this.region}.vtex.io/_account/${this.account}`,
```

Comment:
I think `apps`, `kube-router` and `housekeeper` deserved separate clients as they are actually separate services

Reply:
Considering our case here, I thought it would be good to have routes specific to the sponsor account functionality bundled in the same client. But it is true that this makes the code less maintainable. I will add these functions to the clients in `@vtex/api` in a later PR.